azkaban-common/src/main/java/azkaban/executor/ExecutionLogsCleaner.java

```diff
@@ -0,0 +1,69 @@
+package azkaban.executor;
+
+import azkaban.utils.Props;
+import java.util.concurrent.Executors;
```

Comment:
Thanks for switching to these -- it's much cleaner and concise this way!

Reply:
Thank you for your quick reviews Yuliy!